cmd/clouddriver/clouddriver.go

```diff
@@ -45,24 +46,31 @@ func init() {
 		log.Fatal(err.Error())
 	}
 
+	// Grab our artifact credentials from /opt/spinnaker/artifacts/config.
+	artifactCredentialsController, err := artifact.NewDefaultCredentialsController()
+	if err != nil {
+		log.Println("[CLOUDDRIVER] error setting up artifact credentials controller:", err.Error())
+	}
+
 	sqlClient := sql.NewClient(db)
 	kubeController := kubernetes.NewController()
 	helmClient := helm.NewClient("https://kubernetes-charts.storage.googleapis.com")
 	arcadeClient := arcade.NewDefaultClient()
 
 	arcadeAPIKey := os.Getenv("ARCADE_API_KEY")
 	if arcadeAPIKey == "" {
-		log.Println("WARNING: ARCADE_API_KEY not set")
+		log.Println("[CLOUDDRIVER] WARNING: ARCADE_API_KEY not set")
```

Comment:
Is there a way you could check that the ARCADE_API_KEY is valid?

Reply:
We could probably ask arcade for a token, but I think an empty string check is enough. I think it would be weird to validate a connection to a microservice when we aren't validating other connections at startup. Additionally, we need to allow people to run clouddriver locally, so a warning here is fine.